src/test/java/com/cst438/controller/SectionControllerUnitTest.java

```diff
@@ -42,6 +42,7 @@ public void addSection() throws Exception {
                 2024,
                 "Spring",
                 "cst499",
+                "title",
```

Comment:
Thanks for fixing this!

For some reason it kept giving me issues when I was initially testing it, but mow it works!